src/modules/fancyzones/lib/trace.cpp

```diff
@@ -189,6 +189,7 @@ void Trace::SettingsChanged(const Settings& settings) noexcept
         TraceLoggingBoolean(settings.restoreSize, "RestoreSize"),
         TraceLoggingBoolean(settings.use_cursorpos_editor_startupscreen, "UseCursorPosOnEditorStartup"),
         TraceLoggingBoolean(settings.showZonesOnAllMonitors, "ShowZonesOnAllMonitors"),
+        TraceLoggingBoolean(settings.multiMonitorMode, "MultiMonitorMode"),
```

Comment:
settings.multiMonitorMode and "MultiMonitorMode" -> settings.spanZonesAcrossMonitors and SpanZonesAcrossMonitors